packages/maskbook/src/components/InjectedComponents/DecryptedPost/DecryptedPost.tsx

```diff
@@ -19,6 +19,7 @@ import { Err, Ok } from 'ts-results'
 import { or } from '../../custom-ui-helper'
 import { usePostInfo } from '../../../components/DataSource/usePostInfo'
 import type { Payload } from '../../../utils/type-transform/Payload'
+import { injectPostImageRevealerAtTwitter } from '../../../social-network-provider/twitter.com/ui/injectPostImageRevealer'
```

Comment:
You should not import network provider here. It should be initialized in `activateSocialNetworkUI`